tests/test_django.py

```diff
@@ -16,15 +16,11 @@
 if django is not None:
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'tests.djapp.settings')
 
-    if django.VERSION >= (1, 7, 0):
-        django.setup()
+    django.setup()
     from django import test as django_test
     from django.conf import settings
     from django.db import models as django_models
-    if django.VERSION <= (1, 8, 0):
```

Comment:
Does this mean that ``django.test.runner.DiscoverRunner`` already existed in Django 1.8?